variable.js

```diff
@@ -0,0 +1,58 @@
+const birthYear=1950;
+
+let futureYearplusOne=2001;
```

Comment:
Instead of futureYearplusOne = 2001, we can also make it futureYearplusOne = futureYear + 1. That way we can let program do all the math for us.